public/locales/en/reader.json

```diff
@@ -24,7 +24,7 @@
     "next": "Next",
     "close": "Close",
     "modal-general": {
-        "greeting": "HiRys! Happy 3rd Anniversary!",
+        "greeting": "HiRyS! Happy 3rd Anniversary!",
```

Comment:
This kind of stuff should be stored in the CMS instead. This manga reader might get reused in the future, so having hard-coded strings specific to a project should be avoided. Doesn't need to fixed in this PR, but should be dealt with in another one.